requests-for-comments/2024-08-14-youtuber.md

```diff
@@ -0,0 +1,30 @@
+# Request for comments: Hire a YouTuber
+
+PostHog is built on content + word of mouth. YouTube has proven to be an effective channel for word of mouth grow, based on the significant number of people mention YouTube (or Theo) on signup, but we don’t have the expertise to do content right there. We differentiate by 
+
+YouTube rewards consistency and iteration. Our subscriber base and watch time continues to grow even though we don’t upload more videos. YouTube doesn’t work if you do it part time. You need to go all-in on video. 
+
+I think we have seen some successes, but have been too scattered to really capitalize on them:
+
+- Theo + YouTube mentions shows its promise
+- Although YouTube **tutorials** about PostHog don’t seem to work super well, topic-based (like blogs) YouTube videos do.
+- Although, anecdotally, my small number of YouTube videos get **more** mention than my blogs or newsletters from people, especially when they are learning PostHog. The complexity of PostHog lends itself to video. It is a great way to teach people how to use it.
+
+Beyond YouTube, what could a YouTuber do:
+
+- Create video social content + changelog like so many people have wanted to.
+- Experiment with short form content.
+- Collaborate with influencers (we have done this with newsletters and it works well). This would also help with my influencer plan.
+- Read out 20,000 names (just kidding… unless…).
+
+This does require the **right person**, a proper YouTuber, probably someone who already is one now. I think this is a really cool role for them and PostHog is the right company to attract them. We could talk to the YouTubers we are working with now. 
```

Comment:
Do you see this more as a contractor or a full time properly in house person at PostHog, or do you not have an opinion either way?

Reply:
I'd say in-house full-time, personally. If we're going to do it, we should go all in.

I love this idea. We did something similar to kickstart a YouTube channel back when I worked in publishing and it was one of the best things we ever did.

There is always a danger they'd eventually leave if their own YT does well, but there's way more incentives to work at PostHog than there was to work for a UK publisher with no money!

Let's do this.

Reply:
Additional thought: There's a small chance there could be some overlap here with the Community Manager role, depending on who we hire there. Worth bearing that mind so don't end up doubling up.